client/html/pages/panel/index.html

```diff
@@ -1,123 +1,141 @@
-                <a class="btn btn-icon-toggle">
-                  <i class="md mdi mdi-border-color"></i>
-                </a>
-              </div>
+      </div>
+      <!-- Profile's card -->
+      <div class="col-lg-offset-1 col-lg-3 col-md-4">
+        <div class="card card-underline">
+          <div class="card-head style-primary">
+            <header>Meu perfil</header>
+            <div class="tools">
+              <a class="btn btn-icon-toggle">
+                <i class="md mdi mdi-border-color"></i>
+              </a>
             </div>
-            <div class="card-body no-padding">
-              <ul class="list">
-                <li class="tile">
-                  <div class="tile-content">
-                    <div class="tile-icon">
-                      <img src="{{avatar 40}}" alt="Perfil" />
+          </div>
+          <div class="card-body no-padding">
+            <ul class="list">
+              <li class="tile">
+                <div class="tile-content">
+                  <div class="tile-icon">
+                    <img src="{{avatar 40}}" alt="Perfil" />
+                  </div>
+                  <div class="tile-text">
+                    <a class="ink-reaction" href="{{pathForProfile}}">
+                      {{#if isStudent}}
+                        {{ user.profile.nome }}
+                      {{else}}
+                        {{ user.profile.dados }}
+                      {{/if}}
+                    </a>
+                    <small>Lorem ipsum dolor sit amet, consectetur adipisicing</small>
                     </div>
-                    <div class="tile-text">
-                      <a class="ink-reaction" href="{{pathForProfile}}">
-                        {{#if isStudent}}
-                          {{ user.profile.nome }}
-                        {{else}}
-                          {{ user.profile.dados }}
-                        {{/if}}
-                      </a>
-                      <small>Lorem ipsum dolor sit amet, consectetur adipisicing</small>
-                     </div>
-                   </div>
-                </li>
-              </ul>
-            </div>
+                  </div>
+              </li>
+            </ul>
           </div>
+        </div>
           
-          <div class="card card-underline">
-            <div class="card-head style-primary">
-              {{#if isStudent}}
-                <header>Vagas candidatadas</header>
-              {{else}}
-                <header>Vagas com candidatos</header>
-              {{/if}}
-            </div>
-            <div class="card-body"></div>
+        <div class="card card-underline">
+          <div class="card-head style-primary">
+            {{#if isStudent}}
+              <header>Vagas candidatadas</header>
+            {{else}}
+              <header>Vagas com candidatos</header>
+            {{/if}}
+          </div>
+          <div class="card-body">
+            <ul class="list">
```

Comment:
This block shouldn't be listed as a Profile card, so it should be removed. As we haven't found a way to candidate on jobs yet.